.github/workflows/ci.yml

```diff
@@ -27,10 +27,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        sdk: [stable, 3.4.0, 3.0.0, 2.19.6, 2.18.7, 2.17.7]
-        exclude:
-          - os: windows-latest
-            sdk: 2.17.7
```

Comment:
Is there a reason for removing the other versions? The `pubspec` does specify `'>=2.17.0 <4.0.0'` so it seems reasonable to test different ranges. I'm not involved with Dart/Flutter as much these days so there might something I don't know.